canvas_account_admin_tools/templates/canvas_account_admin_tools/dashboard_account.html

```diff
@@ -58,6 +58,13 @@ <h4 class="list-group-item-heading">Find Course Info</h4>
           </a>
         </div>
 
+        <div class="list-group col-md-4">
+          <a type="button" class="btn btn-primary list-group-item active" href="{% url 'cross_list_courses:index' %}" class="list-group-item active">
+              <h4 class="list-group-item-heading">Cross-list Courses</h4>
+              <p class="list-group-item-text">Set up and modify cross-listings <br>&nbsp;</p>
```

Comment:
What's the `<br>&nbsp;` for? Seems like we should probably achieve whatever that's doing with a CSS style...